Assets/Scripts/PlayerBehaviour.cs

```diff
@@ -53,8 +53,30 @@ void FixedUpdate()
         StartCoroutine(DashBack());
     }
 
-    void DashLeft() {
+    public void CastWindForward() {
+        GameObject newWindSlash = Instantiate(windslash, transform.position + (transform.forward * 2f), transform.rotation, transform);
+        StartCoroutine(DashForward());
+    }
+
+    public void ThrowPlayerBack(float horizontal, float vertical, float duration){
+        StartCoroutine(ThrowBack(horizontal, vertical, duration));
+    }
 
+    IEnumerator DashLeft() {
+        float duration = 0.6f;
+        float startTime = Time.time;
+        float currentTime = (Time.time - startTime) / duration;
+        while (currentTime < 1f) {
+            transform.position -= transform.right * dashSpeed * Time.deltaTime;
+
+            currentTime = (Time.time - startTime) / duration;
+            // print(currentTime);
+            float vertical = (Mathf.Sin(currentTime * Mathf.PI) * dashHeight) + playerHeight;
+
+            transform.position = new Vector3(transform.position.x, vertical, transform.position.z);
```

Comment:
Setting the vertical directly here may end up cause problems if there is something else affecting their y coordinate when this is called. Probably best to factor in the current vertical coordinate